common/src/main/res/values-fr/strings.xml

```diff
@@ -247,7 +247,7 @@ Dans tous les cas, vous devez aussi présenter une pièce d'identité sur demand
     <string name="wallet_faq_works_answer_2">"Oui. Vous pouvez par exemple stocker dans l'app les certificats COVID de tous les membres de votre famille. À noter également dans ce cas qu'un certificat COVID n'est valable que si son détenteur présente une pièce d'identité."</string>
 
     <!-- Antwort 8 im FAQ-Screen der Wallet App -->
-    <string name="wallet_faq_works_answer_3">"L'app contient la date d'expiration de votre certificat COVID en Suisse. En pressant le bouton prévu à cet effet, vous pouvez en outre vérifier dans la vue détaillée si votre certificat COVID est encore valable."</string>
+    <string name="wallet_faq_works_answer_3">"L’app verifica automaticamente se il certificato soddisfa i criteri di validità applicati in Svizzera. Se il certificato è scaduto o se non è valido per motivi tecnici, l’informazione compare direttamente nell’app."</string>
```

Comment:
This should be in french!

Reply:
This issue is fixed with the backend config request, this overrides this file content. We will ship this fallback file again for the release 3.3 with the correct language.

Reply:
Thanks for noticing this. We will fix this in the next release on Monday.